fastlane/lanes/build.rb

```diff
@@ -203,6 +203,50 @@
     end
   end
 
+  #####################################################################################
+  # build_and_upload_installable_build
+  # -----------------------------------------------------------------------------------
+  # Build a WordPress Installable Build and make it available for download
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_upload_installable_build
+  #####################################################################################
+  desc "Build an Installable Build and make it available for download"
+  lane :build_and_upload_wordpress_installable_build do | options |
```

Comment:
`| options |`, _WordPress-style formatting_ 😅

Reply:
`rubocop` would definitively have complained about that one… if we had it set up to lint the `fastlane/lanes/*.rb` files 😅  (one day…)